ethereum/referencetests/build.gradle

```diff
@@ -29,6 +29,37 @@ def blockchainReferenceTests = tasks.register("blockchainReferenceTests")  {
     )
 }
 
+configurations {
+  // we need this because referenceTestImplementation is defined as 'canBeResolved=false'.
+  tarConfig.extendsFrom referenceTestImplementation
+  tarConfig {
+    canBeResolved = true
+    canBeConsumed = false
+  }
+}
+
+def execSpecTests = tasks.register("execSpecTests") {
+  final referenceTestsPath = "$buildDir/tmp/execution-spec-tests/"
+  final generatedTestsPath = "$buildDir/generated/sources/reference-test/$name/java"
```

Comment:
Wondering what the right level is to group these...i.e. should they be alongside the "reference-test" or underneath it as you have here.

What's your thinking on this?

Reply:
I just followed existing definitions....these tasks are basically transformation tasks. they are applying the template to transform the json files to Java files and registering the output paths as `src` for the referenceTest.

Reply:
I guess the question is "is there any value in keeping execution-spec-test separate to reference-test?" 
I suspect not, esp since they are all running under the `referenceTest` command

Reply:
Gary has "extraction" as a seperate task ... I avoided it by using `copy` task within definition of `executionSpecTests`.